commands.go

```diff
@@ -32,12 +32,13 @@ type Command struct {
 
 type commandWrapper struct {
 	label  string
+	writeBeforeFlush bool
```

Comment:
can we name this in a somewhat more meaningfull  way?  I had to go over this a few times before i understood what this name meant ;)